doc/source/whatsnew/v0.18.2.txt

```diff
@@ -186,6 +186,13 @@ Other enhancements
 ^^^^^^^^^^^^^^^^^^
 
 - The ``.tz_localize()`` method of ``DatetimeIndex`` and ``Timestamp`` has gained the ``errors`` keyword, so you can potentially coerce nonexistent timestamps to ``NaT``. The default behaviour remains to raising a ``NonExistentTimeError`` (:issue:`13057`)
+- ``pd.to_numeric()`` now accepts a ``downcast`` parameter (:issue:`13352`)
+
+  .. ipython:: python
+
```

Comment:
also I think we need to update the docs: http://pandas.pydata.org/pandas-docs/stable/basics.html#object-conversion (you can make a new issue for this); as this is outdated (and should now include using ``pd.to_numeric``)